File: typescript/base-hmi-types/src/screens/widgets/HmiLabel.ts
import { HmiTextWidgetBase } from "./HmiTextWidgetBase.js";
import { HmiObjectType } from "../base/HmiObjectType.js";

export class HmiLabel extends HmiTextWidgetBase {
  constructor() {
    super();
    this.hmiObjectType = HmiObjectType.HmiLabel;
  }
}

File: src/BaseHmiTypes/Screens/Widgets/HmiLabel.cs
using BaseHmiTypes.Screens.Base;

namespace BaseHmiTypes.Screens.Widgets;

public class HmiLabel : HmiTextWidgetBase
{
    public HmiLabel()
    {
        HmiObjectType = BaseHmiTypes.Screens.Base.HmiObjectType.HmiLabel;
    }

    public HmiProperty<int>? FieldLength { get; set; }
}
